src/uniparc/config/UniParcColumnConfiguration.tsx

```diff
@@ -152,14 +154,24 @@ UniParcColumnConfiguration.set(UniParcColumn.protein, {
   ),
 });
 
+type XRefWithProteomeId = UniParcXRef &
+  Required<Pick<UniParcXRef, 'proteomeId'>>;
 UniParcColumnConfiguration.set(UniParcColumn.proteome, {
   label: 'Proteomes',
-  render: (data) => (
+  render: ({ uniParcCrossReferences }) => (
     <ExpandableList descriptionString="proteomes" displayNumberOfHiddenItems>
-      {xrefGetter(data, 'proteomeId')?.map((accession) => (
-        <Link key={accession} to={getEntryPath(Namespace.proteomes, accession)}>
-          {accession}
-        </Link>
+      {uniqBy(
+        uniParcCrossReferences?.filter((xref): xref is XRefWithProteomeId =>
+          Boolean(xref.proteomeId)
+        ) || [],
+        (xref) => `${xref.proteomeId}-${xref.component}`
+      ).map((xref) => (
+        <Fragment key={`${xref.proteomeId}-${xref.component}`}>
+          <Link to={getEntryPath(Namespace.proteomes, xref.proteomeId)}>
+            {xref.proteomeId}
+          </Link>
+          {xref.component ? `:${xref.component}` : undefined}
```

Comment:
Formatting: would add a space after ':'

Reply:
I'm matching the current behaviour of the website, that's why I didn't put a space.
How should a user read the information of this cell with the component information? I'm not sure what I'm seeing here, that's why I had followed the current website's implementation

Reply:
For proteomes, "component" means either whole genome (bacterial genome), a chromosome (chromosome 21) or mitochondria (I think that's it). I think this is the list of components and associated proteomes, and they just used a semicolon as a separator and forgot to add a space.

Reply:
There's not much information on the website about those https://www.uniprot.org/help/proteome_component
I'm wondering in the first place why it was represented like this and not as `<Proteome ID> (<component>)` which looks more like a way to specify some extra information in plain English rather than the `:` without extra spaces which I read as some kind of accession specifier (hence why I was keeping the exact same syntax, just in case...)

Reply:
Ah, sorry, crossed message.
I do see some "exotic" component names some times, not sure if that's normal
![Screenshot 2021-03-02 at 17 15 29](https://user-images.githubusercontent.com/6025561/109687279-ff09f000-7b7a-11eb-9941-5611856481f6.png)

Reply:
I think the parenthesis would be nicer indeed.

Reply:
> Ah, sorry, crossed messages.
> I do see some "exotic" component names some times, not sure if that's normal
> ![Screenshot 2021-03-02 at 17 15 29](https://user-images.githubusercontent.com/6025561/109687279-ff09f000-7b7a-11eb-9941-5611856481f6.png)

Yep, plasmids are circular bits of DNA circulating in Bacterias (but I guess you know that already...) and the Unassembled are from shotgun sequencing (bits they couldn't match to anything)